recipes/gnali/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
Please remove this build script and add this to the `build:` section in the recipe:
```
script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
```

Reply:
Done, thank you for looking at my pull request and letting me know!